cmd/root.go

```diff
@@ -0,0 +1,148 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/adrg/xdg"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"sigs.k8s.io/hydrophone/pkg/client"
+	"sigs.k8s.io/hydrophone/pkg/common"
+	"sigs.k8s.io/hydrophone/pkg/service"
+)
+
+var (
+	cfgFile          string
+	kubeconfig       string
+	parallel         int
+	verbosity        int
+	outputDir        string
+	cleanup          bool
+	listImages       bool
+	conformance      bool
+	focus            string
+	skip             string
+	conformanceImage string
+	busyboxImage     string
+	dryRun           bool
+	testRepoList     string
+	testRepo         string
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "hydrohpone",
+	Short: "Hydrophone is a lightweight runner for kubernetes tests.",
+	Long:  `Hydrophone is a lightweight runner for kubernetes tests.`,
+	PreRun: func(cmd *cobra.Command, args []string) {
+		if conformance && focus != "" {
```

Comment:
this can be implemented using `rootCmd.MarkFlagsMutuallyExclusive("conformance", "focus") `to mark flags as mutually exclusive